### ISSUE TITLE
Add attributes to provide button labels, and button aria labels

### ISSUE THREAD

**Issue by kpander** [kpander/carousel-js#4]
We should be able to easily specify the button label (and screenreader label). We shouldn't have to provide a custom template id to do that.

Example potential implementation:

```html
<carousel-ithreads 
  label-next="&raquo;" 
  aria-next="Next item" 
  label-previous="&laquo;" 
  aria-previous="Previous item"
></carousel>
```

... should generate buttons in the carousel component similar to:

```html
<button id="btnNext" aria-label="Next item">&raquo;</button>
```

If an `aria-*` attribute is not specified, no `aria-label` attribute should be created.

If a `label-*` attribute is not specified, use the default labels (e.g., "Next", "Previous").
